ui/app/src/state/epics/dialogs.ts

```diff
@@ -125,6 +130,58 @@ const dialogEpics: CrafterCMSEpic[] = [
           catchAjaxError(fetchDeleteDependenciesFailed)
         )
       )
+    ),
+  (action$, state$, { getIntl }) =>
+    action$.pipe(
+      ofType(showEditDialog.type),
+      withLatestFrom(state$),
+      switchMap(([{ payload }, state]) => {
+        if (payload.src === state.dialogs.edit.src) {
+          const id = 'legacy-editor';
+          if (state.dialogs.minimizedDialogs[id].minimized === true) {
+            return of(
+              updateDialog({
+                id,
+                minimized: false
+              })
+            );
+          } else {
+            return NEVER;
+          }
+        } else {
+          return of(
+            showConfirmDialog({
+              body: getIntl().formatMessage(formEngineMessages.inProgressConfirmation)
+            })
```

Comment:
The confirm dialog presents a question and this isn't one. We need to change the graphic and only the accept button should show up. See Confirm Dialog designs on Figma (3.2 project) for guidance.

We need to tweak ConfirmDialog to allow graphic swapping.
```jsx
function ConfirmDialogWrapper(props: ConfirmDialogProps) {
  const { onOk, onCancel, body, title, children, classes, imageUrl = questionGraphicUrl } = props;
  const { formatMessage } = useIntl();
  useUnmount(props.onClosed);
  return (
    <>
      <DialogContent id="confirmDialogBody" className={classes.dialogBody}>
        <img src={imageUrl} alt="" />
```